.github/workflows/generate-exploratory.yml

```diff
@@ -39,70 +26,78 @@ on:
         type: string
         required: false
         description: Filename containing markdown content from previous regression step
+    secrets:
+      key:
+        required: true
+      GH_TOKEN:
+        required: true
+      LOGIN_USERNAME:
+        description: The username to use for login
+      LOGIN_PASSWORD:
+        description: The password to use for login
     outputs:
       pr-branch:
         value: ${{ jobs.exploratory.outputs.pr-branch }}
 
 jobs:
   exploratory:
+    name: Generate Exploratory Test Plans - Depth ${{ inputs.depth }}/${{ inputs.max-depth }}
     runs-on: ubuntu-latest
     outputs:
       pr-branch: ${{ steps.create-pr-branch.outputs.pr-branch }}
     steps:
-      - name: Check recursion depth
-        run: |
-          if [ ${{ inputs.depth }} -ge 3 ]; then
-            echo "Max recursion depth reached. Exiting."
-            exit 0
-          fi
-
       - id: create-pr-branch
         name: Create PR branch name
         shell: bash
         run: echo "pr-branch=testdriver/exploratory-${{ github.run_id }}-${{ github.run_attempt }}" >> $GITHUB_OUTPUT
 
+      - uses: 'actions/github-script@v6'
+        id: create-prompt
+        env:
+          DEPTH: ${{ inputs.depth }}
+          PREVIOUS_FILE: ${{ inputs.previous-file }}
+          LOGIN_USERNAME: ${{ secrets.LOGIN_USERNAME }}
+          LOGIN_PASSWORD: ${{ secrets.LOGIN_PASSWORD }}
+        with:
+          script: |
+            const depth = parseInt(process.env.DEPTH)
+            const username = process.env.LOGIN_USERNAME
+            const password = process.env.LOGIN_PASSWORD
+            let loginFile = depth === 1 && username && password ? 'login.yml' : ''
+            let previousFile = loginFile || process.env.PREVIOUS_FILE
+
+            let instructions = [
+              `/generate web 3${previousFile ? ` testdriver/${previousFile} --skip-yaml` : ''}`,
```

Comment:
@yassine-cc this is interesting `--skip-yaml`. I feel like that's not supported?